htdocs/core/lib/functions.lib.php

```diff
@@ -2761,7 +2761,7 @@ function dol_mktime($hour, $minute, $second, $month, $day, $year, $gm = 'auto',
 	}
 	//var_dump($localtz);
 	//var_dump($year.'-'.$month.'-'.$day.'-'.$hour.'-'.$minute);
-	$dt = new DateTime(null, $localtz);
+	$dt = new DateTime('', $localtz);
```

Comment:
'' seems not valid.  Is it ok with  'now'  (suggested by php doc)  ?

Reply:
@eldy this is the warning : 
datetime::__construct(): passing null to parameter #1 ($datetime) of type string is deprecated